### ISSUE TITLE
Learnyounode doesn't run (Node.js 4.3.2 and Win 10)

### ISSUE THREAD

**Issue by EmppuTheEngineer** [workshopper/learnyounode#421]
Yep, another one, since restarting neither node.js or computer didn't worked and google really didn't give me other solutions.

So, I've installed learnyounode via command line, however, it won't run. While I write 
learnyounode on the command line, it will return the following;

```
'learnyounode' is not recognized as an internal or external command,
operable program or batch file.

```
I have installed both node.js and learnyounode (apparently successfully, since node.js works with other project and I can find the learnyounode-files on node_modules). I have also tried to navigate to node_modules directory to start learnyounode from there, but no success so far.

And I have googled this issue, yet it only brought me such solutions where learnyounode worked fine after restarting either computer or node.js, which didn't really work for me.

Thanks beforehand!

**Comment by MystK** [comment]
Hello, it works fine for me. Are you sure you ran the command: `npm install learnyounode -g`

**Comment by martinheidegger** [comment]
@EmppuTheEngineer Have you found a way?

**Comment by pramodvspk** [comment]
It doesnt work for me too, how to make this work?

**Comment by MystK** [comment]
Guys, in order to get help, you have to help us help you. Did anyone try running the command I put?

**Comment by victoray** [comment]
You need to install it with `-g` option (it's mean you will install `learyounode` in `global` environment).

**Comment by idietmoran** [comment]
`npm install -g learnyounode` does not work on windows 10, then menu / arrow keys do not work on either `cmd.exe` or `powershell.exe`

It works find on linux though, I assume it's a bug with windows.

**Comment by MystK** [comment]
Do you have Node installed? If npm install doesn't work, you don't have Node installed.

**Comment by idietmoran** [comment]
@MystK yes ofc i have node installed, it arrow keys do not work on the menu, and pressing enter creates a new line outside of the menu screen. 

As I said it works find on linux, but on windows it's bugged.

**Comment by MystK** [comment]
@idietmoran, it sounds like npm install -g learnyounode does work and you do not have the same issue as this issue...

**Comment by idietmoran** [comment]
@MystK I must've commented in the wrong issue, sorry.

**Comment by boggen** [comment]
windows 10, node 6.2 (cmd.exe, cmd.exe w/ admin rights, git bash prompt) all 3 same thing as below.

========================
C:\Users\me\AppData\Roaming\npm\node_modules> npm install learnyounode -g
C:\Users\me\AppData\Roaming\npm\node_modules> cd learnyounode
C:\Users\me\AppData\Roaming\npm\node_modules\learnyounode>node learnyounode.js
(empty space)
C:\Users\me\AppData\Roaming\npm\node_modules\learnyounode>

==========================
not sure were to look for any sort of log files or commands to output errors.

**Comment by martinheidegger** [comment]
@boggen I don't quite know what your problem is but I can offer you to contact me via gitter: https://gitter.im/martinheidegger to arrange for a debug session?!

**Comment by arebee** [comment]
The documentation is broken in the Windows install.
The following worked for me:
- install using the -g switch
- from the Node.js Command Prompt and the plain command prompt both of these work:
 - ```c:\Users\<yourUserName>\AppData\Roaming\npm\learnyounode.cmd```
 - ```learnyounode``` (takes a while, hunts on the path)

```node learnyounode``` fails every time for me.
Arrows in the cmd prompt work - Microsoft Windows [Version 10.0.14393]
